sdk/python/packages/flet-core/src/flet_core/view.py

```diff
@@ -139,8 +139,8 @@ def controls(self) -> List[Control]:
         return self.__controls
 
     @controls.setter
-    def controls(self, value: List[Control]):
-        self.__controls = value
+    def controls(self, value: Optional[Sequence[Control]]):
```

Comment:
Same here.

Reply:
It seems the wrong one was selected here; it should be another file.